Runtime/Random/IRandomString.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2023 Koji Hasegawa.
+// This software is released under the MIT License.
+
+namespace TestHelper.Monkey.Random
+{
+    public interface IRandomString
```

Comment:
I think I will choose migration if it can restrict character kind in generated string.

Reply:
Thank you.
Considering availability, it may be better to keep IRandomString independent and pass the implementation for each TextField.name.

e.g.,
```
public Dictionary<string, IRandomString> = new Dictionary(){
  { testFIeldName: "userid field", new UserIdRandomString() },
  { testFIeldName: "password field", new PasswordRandomString() },
}
```

Reply:
Ah, it looks sweet. But name-based strategy might not work some conditions. So I suggest `Func<GameObject, IRandomString>` to keep extensibility.